samcli/commands/delete/delete_context.py

```diff
@@ -69,22 +82,28 @@ def parse_config_file(self):
             if not self.stack_name:
                 self.stack_name = config_options.get("stack_name", None)
             # If the stack_name is same as the one present in samconfig file,
-            # get the information about parameters if not specified by customer.
+            # get the information about parameters if not specified by user.
             if self.stack_name and self.stack_name == config_options.get("stack_name", None):
                 LOG.debug("Local config present and using the defined options")
                 if not self.region:
                     self.region = config_options.get("region", None)
-                    click.get_current_context().region = self.region
+                    Context.get_current_context().region = self.region
```

Comment:
do we need to double check that the region we get it from the config file is not none before updating the context ?

Reply:
That makes sense, context should be updated after profile and region are assigned values other than None.